scripts/zones/Western_Adoulin/npcs/Ansegusele.lua

```diff
@@ -0,0 +1,53 @@
+-----------------------------------
+--  Area: Western Adoulin
+--  NPC: Ansegusele
+--  Type: Shop NPC
+--  @zone 256
+--  @pos 74 4 -126 256
+-----------------------------------
+package.loaded["scripts/zones/Western_Adoulin/TextIDs"] = nil;
+-----------------------------------
+require("scripts/globals/shop");
+require("scripts/zones/Western_Adoulin/TextIDs");
+
+local stock =
```

Comment:
This should be inside the function its being used in, down between lines 37 and 38.